pythonforandroid/bootstraps/service_only/build/templates/Service.tmpl.java

```diff
@@ -17,7 +17,7 @@
      * {@inheritDoc}
      */
     @Override
-    public int getStartType() {
+    public int startType() {
```

Comment:
I think I've seen this change breaking things in the past. I haven't tried on device (Android 7) yet, but are we sure-ish about it?